package test.paragraph3.empty_block

fun foo () {
    try {
        doSome()
    } catch (ex: Exception){
}
}